### PR TITLE
More prominent styling on Finalize Arrangement button

### PR DESCRIPTION
**Pull request by eckardm**

### CODE REVIEW

File: app/front_page/content.html [hunk 1]
```diff
@@ -219,7 +219,7 @@
                ng-click="remove(selected)"
                value='Delete selected'>
         <input type='button'
-               class='btn btn-primary btn-sm'
+               class='btn btn-primary btn-sm btn-success'
```
Comment (sevein) [inline app/front_page/content.html]: Are you not removing `btn-primary` on purpose? It seems to be mutually exclusive with `btn-success`.

Comment (eckardm) [reply app/front_page/content.html]: Ah, that slipped by--thanks for letting me know. I'll update and resubmit.

**Comment by sevein** [comment]
Great, thank you! Can you squash the three commits into one?

**Comment by eckardm** [comment]
Squashed!

**Pull request merged**